src/components/List/List.js

```diff
@@ -0,0 +1,52 @@
+import Bubbles from "../Bubbles";
+import usePageData from "../../hooks/usePageData";
+import ButtonAppBar from "../AppBar";
+import FAB from "../FAB";
+import { useState } from "react";
+import AddCard from "./AddCard";
+import { useLocation } from "@reach/router";
+import Skeleton from "@material-ui/lab/Skeleton";
+
+function List() {
+  const [update, setUpdate] = useState(false);
+  const [list, isLoading] = usePageData("ToDo", update);
+  const [openAddCard, setOpenAddCard] = useState(false);
+  const location = useLocation();
+
+  let listName = "";
+  let cardTitle = "";
+  if (location.pathname === "/clean") {
+    listName = "Putzen";
+    cardTitle = "Haushaltsaufgabe";
+  } else if (location.pathname === "/todo") {
+    listName = "ToDo";
+    cardTitle = "Aufgabe";
+  } else if (location.pathname === "/shopping") {
+    listName = "Einkaufen";
+    cardTitle = "Artikel";
+  }
+
+  if (isLoading) {
+    return <Skeleton variant="rect" height={48} animation="wave" />;
```

Comment:
```suggestion
    return <Skeleton variant="text" animation="wave" />;
```
Das soll doch die Liste darstellen oder? Dann macht doch der Textbalken mehr Sinn oder?